src/include/daos_srv/dtx_srv.h

```diff
@@ -156,6 +176,23 @@ enum dtx_status {
 	DTX_ST_PREPARED		= 1,
 	/** The DTX has been committed. */
 	DTX_ST_COMMITTED	= 2,
+	/** The DTX is committable, but not committed. */
+	DTX_ST_COMMITTABLE	= 3,
+	/** The DTX is corrupted, some participant RDG(s) may be lost. */
+	DTX_ST_CORRUPTED	= 4,
+	/** The DTX is in-resync, not sure its status. */
+	DTX_ST_UNCERTAIN	= 5,
```

Comment:
Probably a different term here too.

Reply:
Then "DTX_ST_RESYNC" ?

Reply:
I found the solution to avoid "UNCERTAIN" name. I will do that if have chance to refresh the patch; otherwise, it will be in subsequent DTX patch.

Reply:
It is done in the patch PR#3939. @jolivier23